crates/libs/bindgen/src/types/class.rs

```diff
@@ -83,7 +81,7 @@ impl Class {
             }
         );
 
-        let factories = required_interfaces.iter().filter_map(|interface| match interface.kind {
+        let factories = required_interfaces.                 iter().filter_map(|interface| match interface.kind {
```

Comment:
I also noticed that `cargo fmt` seems to give up on some larger closures. Not sure what's going on there...